pmacApp/pmc/trajectory_scan_definitions_ppmac.pmh

```diff
@@ -1,8 +1,8 @@
 // *****************************************************************************************
 // Variables
 // *****************************************************************************************
-#define BuffLen       1000    // Length of buffers
-#define DoubleBuffLen 4000
+#define BuffLen        500    // Length of buffers
```

Comment:
Worth a comment to say why 500, especially as the DoubleBuffLen variable name doesn't fit anymore with the value change

Reply:
The variable name didn't fit before this change, I've just adjusted the values keeping the same ratio, but I agree that is confusing.

I will review that. I suspect that is related to incorrect address definition of the BufferB
```
// Set Initial Values
BufferLength = BuffLen                          // BuffLen defined in header file
BufferAdr_A = 0                                 // BufferAdr defined in header file
BufferAdr_B = 2 * BufferLength                  // each individual array holds 2 buffers
```
https://github.com/DiamondLightSource/pmac/blob/00568ed39924bc5af541a8fb5cf7ea871cc899f4/pmacApp/pmc/trajectory_scan_code_ppmac.pmc#L6C1-L8C89

Reply:
Solved by 33bda474ff6b00281a9722b4a5253bbf236b7cb1

I have confirmed that as it was:
```
#define BuffLen        500    // Length of buffers
#define DoubleBuffLen 2000    // <= 2864 to prevent P-variables/global variables overflow
```
```
BufferAdr_B = 2 * BufferLength
```
only 2000 P-variables were being reserved, but only 1000 were being actually used.
Each one of the buffers was having the last 500 positions skipped.

That was a problem that I have fixed on the PowerPMAC project, and forgot to propagate into the pmac driver files.
Thanks for raising this.